packages/@react-aria/button/test/useButton.test.js

```diff
@@ -28,5 +28,16 @@ describe('useButton tests', function () {
     expect(result.current.buttonProps['aria-disabled']).toBeUndefined();
     expect(result.current.buttonProps.href).toBeUndefined();
     expect(typeof result.current.buttonProps.onKeyDown).toBe('function');
+    expect(result.current.buttonProps.rel).toBeUndefined();
+  });
+  it('handles the rel attribute on anchors', function () {
+    let props = {elementType: 'a', rel: 'noopener noreferrer'};
```

Comment:
I might be missing something about rel, but i tried adding this to our storybook, I'm not seeing noopener come through on the rendered link though, only noreferrer
could you add a story to your PR to demonstrate it in a browser as well?

Reply:
just something like 
```

  .add(
    'element: a, href: \'//example.com\', rel=\'noopener noreferrer\'',
    () => render({elementType: 'a', href: '//example.com', variant: 'primary', rel: 'noopener noreferrer'})
  )
```

Reply:
Done.